docs/en/operation/Metrics-System.md

```diff
@@ -120,9 +125,208 @@ Metrics in Alluxio have the following format for non-master node metrics:
+| Master.getPinnedFileIds | The duration statistics of getting the ids of pinned files 
+| Master.getWorkerId | The duration statistics of getting worker id |
+| Master.registerWorker | The duration statistics of registering worker to master |
+
+* Other Master metrics
+
+| Metric Name | Description |
+|---------------------------------|-----------------------------------------------------|
+| Master.LastBackupEntriesCount | The total number of entries written in last leading master metadata backup |
+| Master.BackupEntriesProcessTime | The process time of the last backup |
+| Master.LastBackupRestoreCount | The total number of entries restored from backup when a leading master initializes its metadata |
+| Master.BackupRestoreProcessTime | The process time of the last restore from backup |
+
+### Worker Metrics
+
+| Metric Name | Description |
+|---------------------------------------------|-----------------------------------------------------|
+| Worker.<WORKER_HOSTNAME>.CapacityTotal | Total capacity of this worker in bytes
+| Worker.<WORKER_HOSTNAME>.CapacityUsed | Used capacity of this worker in bytes
+| Worker.<WORKER_HOSTNAME>.CapacityFree | Free capacity of this worker in bytes
+| Worker.<WORKER_HOSTNAME>.BlocksCached | Total number of blocks in Alluxio worker storages |
+| Worker.<WORKER_HOSTNAME>.BlocksAccessed | Total number of times blocks in this worker are accessed |
+| Worker.<WORKER_HOSTNAME>.BlocksCanceled | Total number of aborted temporary blocks |
+| Worker.<WORKER_HOSTNAME>.BlocksDeleted | Total number of deleted blocks |
+| Worker.<WORKER_HOSTNAME>.BlocksEvicted | Total number of blocks removed by this worker |
+| Worker.<WORKER_HOSTNAME>.BlocksLost | Total number of lost blocks |
+| Worker.<WORKER_HOSTNAME>.BlocksPromoted | Total number of blocks moved by clients from one location to another in this worker |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheRequests | Total number of async cache requests |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheDuplicateRequests | Total number of duplicate requests of caching the same block asynchronously |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheSucceededBlocks | Total number of blocks succeed in async cache |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheFailedBlocks | Total number of blocks failed to be cached asynchronously |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheUfsBlocks | Total number of async cache blocks which have local source |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheRemoteBlocks | Total number of remote blocks to async cache locally |
+
+### Process Common Metrics
+
+The following metrics are collected on each instance (Master, Worker or Client).
+
+* JVM attributes
+
+| Metric Name | Description |
+|-------------------------|-----------------------------------------------------|
+| name | The name of the JVM |
+| uptime | The uptime of the JVM |
+| vendor | The current JVM vendor |
+
+* Garbage collector statistics
```

Comment:
These are default JVM metrics collected, is there a resource we can also point users to for more details?

Reply:
No, I didn't find any docs. Many docs just list out the metrics without description.